BodoSQL/bodosql/context.py

```diff
@@ -765,19 +766,19 @@ def _compile(self, sql, params_dict=None, dynamic_params_list=None):
             False,  # We need to execute the code so don't hide credentials.
         )
         if bodo.get_rank() == 0:
-            is_dll = generator.isDDLProcessedQuery()
+            is_ddl = JavaEntryPoint.isDDLProcessedQuery(generator)
```

Comment:
My hope is the `JavaEntryPoint` can be a standard for understanding an API moves to Java, at least for low level operations.